src/easylink/pipeline_schema.py

```diff
@@ -49,19 +49,13 @@ def validate_inputs(self, input_data: Dict[str, Path]) -> Optional[List[str]]:
         errors = {}
         for _, _, edge_attrs in self.graph.out_edges("input_data", data=True):
             validator = edge_attrs["input_slot"].validator
-            slot_name = edge_attrs["output_slot"].name
-            try:
-                file = input_data[slot_name]
-            except KeyError:
-                errors[str(slot_name)] = ["Missing required input data"]
-                continue
-            if not file.exists():
-                errors[str(file)] = ["File not found."]
-                continue
-            try:
-                validator(file)
-            except Exception as e:
-                errors[str(file)] = [e.args[0]]
+            for file in input_data.values():
```

Comment:
I simplified this, since we don't know the keys at this stage. I suppose another option would be to wait until the pipeline graph has been created to validate the input data, using the output slot names as before.

Reply:
it is simpler, but does it guarantee the same handling given that there is not change of KeyError like before? Would the same check basically be to assert that input_data.values() is non-empty?

Reply:
> Would the same check basically be to assert that input_data.values() is non-empty?
Yes, and this is a good idea

Reply:
done